javalang/tokenizer.py

```diff
@@ -116,6 +116,8 @@ class Operator(JavaToken):
     ASSIGNMENT = set(['=', '+=', '-=', '*=', '/=', '&=', '|=', '^=', '%=',
                       '<<=', '>>=', '>>>='])
 
+    LAMBDA = set(['->'])
```

Comment:
I don't think the '->' operator needs a corresponding set and "is_lambda" method. Since there is only one operator of this type a `token.value == "->"` would suffice. This is also consistent with how the '...' pseudo-operator is being treated.